.rubocop.yml

```diff
@@ -1,5 +1,8 @@
+require:
+  - rubocop-rails
+
 AllCops:
-  TargetRubyVersion: 2.3
+  TargetRubyVersion: 2.5
```

Comment:
Foreman needs ruby min version 2.3

Reply:
See https://github.com/theforeman/foreman/blob/develop/.rubocop.yml <- this was updated to 2.5